recipe/meta.yaml

```diff
@@ -39,53 +48,67 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
     - setuptools >=65
     - setuptools-scm >=6.2
     - pip
   run:
-    - python >=3.9
-    - astropy
+    - python >={{ python_min }}
+    - astropy-base
     - boost-histogram
-    - ctapipe
+    - ctapipe-base
     - eventio
     - jsonschema
     - matplotlib-base
     - numpy
+    - particle
     - pymongo
     - pyproj
     - python-dotenv
     - pyyaml
+    - pytest
```

Comment:
pytest is a *runtime* dependency?

Reply:
At this point yes. We have simtools.testing modules which does provide service functions for the extensive integration tests. But you are right, they are not needed for running simtools and we should move that code into the tests directory. I'll open an issue in simtools, but for now it is needed.